.github/workflows/fuzzing.yml

```diff
@@ -2,6 +2,11 @@ name: fuzzing
 
 on:
   push:
+    # This workflow is disabled.
+    # See #6681 for details. Also, this comment
+    # https://github.com/tarantool/tarantool/pull/6764#issuecomment-1004616730
+    banches-ignore:
+      - '**'
```

Comment:
Typo: banches -> branches.

AFAIR, github actions documentation states that `branches-ignore` and `branches` could not be used simultaneously.

Reply:
I can suggest to use `if` clause in the job (right below `fuzzing:`, before `runs-on:`).

Reply:
Fixed.

Reply:
You disabled a step, not a job. I guess it'll fail.

BTW, I found why the workflow is not run in the pull request: https://github.com/tarantool/tarantool/pull/6758#issuecomment-1004701258.

Reply:
I moved the check to the proper place. We should definitely re-evaluate how the label `full-ci` is working.

Reply:
Now it looks for me as a working solution, but I didn't verify it.

Sure, we should follow up PR #6758.